.github/workflows/tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

Comment:
```suggestion
on:
  pull_request:
  push:
    branches: [ main ]
```

Reply:
It means "launch the CI on every PR commits and for every push on `main`".
Your previous statement excluded the PRs that are not to `main`, which happens in our process sometimes during the pre-release week 🙂